<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamPasser</class>
 <widget class="QWidget" name="ExamPasser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="screen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>911</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="AnswersLayout"/>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_questionText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Next">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>680</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_examName">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Examination Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_question">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>470</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Question 1/10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_scores">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>580</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scores 0/20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_isPassed">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>250</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Passed!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
